modules/jcommunity/locales/en_US/password.UTF-8.properties

```diff
@@ -56,16 +56,15 @@ change.cmdline.error.module = This user uses another authentication module, we c
 change.cmdline.error.defined = The password of this user is already defined, please use the '--force' flag to override it.
 change.cmdline.error.change = An unknown error happened during password change.
 change.cmdline.success = Password changed successfully.
-change.cmdline.help = usage: php cmdline.php jcommunity~reset_pass:change [--force] login [password]\
```

Comment:
you delete this string, this is good. But now I guess command line script for jelix 1.6 are broken, since you forgot to modify these scripts to use the new locale property. Please modify these scripts.